### ISSUE TITLE
Want to clarify about WG persistent connection

### ISSUE THREAD

**Issue by fakemanhk** [triffid/pia-wg#12]
From your note you mentioned that PIA servers reboot occasionally, so it means that if I am running a 24x7 connections it will drop and won't come back due to config change? I am considering to place it on the network with IoTs that really needs to have outgoing internet, but if it will reboot and connections can't come back this will be an issue.

**Comment by triffid** [comment]
Yeah, the setup on PIA's end is ephemeral and will vanish if unused for several hours or if they reboot their server ("every few months" according to PIA support), leaving your end casting packets into the void to be ignored.

Best workaround I can think of is to run the script's reconnect if you detect outage or just periodically from cron or whatever - some router firmwares allow this sort of thing.

Alternatively, set up your own VPS so the remote wireguard config doesn't have to be ephemeral, they're pretty cheap these days
